src/NzbDrone.Common/Options/HttpOptions.cs

```diff
@@ -0,0 +1,12 @@
+namespace NzbDrone.Common.Options;
+
+public class HttpOptions
+{
+    public string UrlBase { get; set; }
+    public string BindAddress { get; set; }
+    public int? Port { get; set; }
+    public bool? EnableSsl { get; set; }
```

Comment:
I think these should either be moved to `HttpsOptions` or `HttpOptions` needs to be renamed to something that is more generic and encompasses HTTPS things.

Reply:
Perhaps `ServerOptions`?

Reply:
I think that works, was the first one that came to mind for me too

Reply:
I've opted to go with ServerOptions 👍